charts/templates/statefulset.yaml

```diff
@@ -21,6 +21,45 @@ spec:
       affinity: {{- toYaml . | nindent 8}}
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
+      {{- if ne .Values.bootstrapHostName "" }}
+      initContainers: 
+      - name: configure-group
+        image: "{{ .Values.initContainerImage.repository }}:{{ .Values.initContainerImage.tag }}"
+        command:
+          - sh
+          - '-c'
+          - |
+            echo '### Begin configuring group ###'
+
+            GROUP_CFG_TEMPLATE='{"group-name":"%s"}'
+            GROUP_CFG=$(printf "$GROUP_CFG_TEMPLATE" "$MARKLOGIC_GROUP")
+
```

Comment:
We should add one more check to see if the bootstrapHostName provided by user is valid.

Reply:
added check for bootstrapHostName

Reply:
The current implementation adds one extra nested if statement to whole logic. Can you please put the logic that checks for valid host name like this:
if host name is not valid, then
echo hostname provided is not valid.

then the following command.